rel/config/config.exs

```diff
@@ -51,7 +51,7 @@ defmodule Utils do
       |> Application.spec(:vsn)
       |> case do
         nil -> Mix.Project.config()[:version]
-        version -> List.to_string(version)
+        version -> String.Chars.to_string(version)
```

Comment:
I think we can even use the `to_string/1`macro directly!

https://hexdocs.pm/elixir/Kernel.html#to_string/1